app/utils/error_handling.ts

```diff
@@ -23,8 +24,9 @@ import {logWarning} from './log';
 class JavascriptAndNativeErrorHandler {
     initializeErrorHandling = () => {
         initializeSentry();
-        setJSExceptionHandler(this.errorHandler, false);
 
+        // NOTE: The below methods have been disabled as they interferred with Sentry error reporting.
```

Comment:
The below methods have been disabled as they interferred with Sentry error reporting.

Reply:
We used the js handler to forward the exception to sentry, why don't we need that anymore?

The native handler uses a third parameter that if you set as true, then the sentry error handler should work just fine.

Perhaps read the README of the library and inspect its source code to understand more

Reply:
reverted